app/views/dashboard/index.html.haml

```diff
@@ -34,6 +34,14 @@
     = render partial: "dashboard/ng-partials/widgets/schedule_sequence"
   %script{:id => "widgets/calendar", :type => "text/ng-template"}
     = render partial: "dashboard/ng-partials/widgets/calendar"
+  %script{:id => "widgets/move", :type => "text/ng-template"}
```

Comment:
You probably should be doing Rails style partial rendering for these. ng-partials come at a bit of a performance cost (each one is a seperate network request and requires the browser to do additonal rendering / computation).

You could just as easily `= render partial: "dashboard/ng-partials/widgets/calendar"` or `<%= render partial: "dashboard/ng-partials/widgets/calendar" %>` and avoid the performance / maintenance overhead of relying on angular to do this.

The main reason I was using ng-partials in the other examples was:
1. The app is an SPA and needs a way of registering each "screen" as a seperate page within angular
2. The sequence steps have no way knowing if they need to be rendered until *after* the page loads.

If you're doing straight copy/paste for organization purposes, it's better to just do server side rendering.